response.go

```diff
@@ -35,6 +80,16 @@ type SQLInfo struct {
 	InfoAutoincrementIds []uint64
 }
 
+// Header is a response header.
+type Header struct {
+	// RequestId is an id of a corresponding request.
+	RequestId uint32
+	// Code is a response code.
+	// Code == OkCode if there is no error.
+	// Otherwise, it contains an error code.
+	Code int
```

Comment:
I was thinking about this question. From a user's point of view, this is not the server response code as is, but an error.

```suggestion
	// Error is a response error. It could be used to check that response have or haven't an error without decoding.
	// Error == ErrorNo (iproto.ER_UNKNOWN) if there is no error.
	// Otherwise, it contains an error code from iproto.Error enumeration.
	Error iproto.Error
```

So it would be more clear to do the change.

```Go
resp, err := future.GetResponse()
if resp.Header().Error != OkCode {
     errCode = iproto.Error(resp.Code)
     fmt.Println("response error code: %s", errCode)
}
// continue
```
vs
```Go
resp, err := future.GetResponse()
if resp.Header().Error != ErrorNo {
     fmt.Println("response error code: %s", resp.Header().Error)
}
// continue
```

Reply:
Updated.